components/dialog/stories/dialog.mdx

```diff
@@ -0,0 +1,64 @@
+import { ArgTypes, Canvas, Meta, Description, Title } from '@storybook/blocks';
+
+import * as DialogStories from './dialog.stories';
+
+<Meta of={DialogStories} title="Docs" />
+
+<Title of={DialogStories} />
+<Description of={DialogStories} />
+
+## Variants
+
+### Small
+
+<Canvas of={DialogStories.Small} />
+
+### Medium (default)
+
+<Canvas of={DialogStories.Default} />
+
+### Large
+
+<Canvas of={DialogStories.Large} />
+
+### Dismissible
+
+A dialog that can be dismissed without taking an actions. Dismissible dialogs should never have buttons.
```

Comment:
Should all/most of the story descriptions / notes in this MDX file be added to the stories file descriptions instead?

Reply:
I can get them over there. They're in a weird order (at least to me), but that might be ok for now. https://github.com/adobe/spectrum-css/pull/2833/commits/2cb52cbd0a6373d5963c13d96b8ac856b5cda1fa